### PR TITLE
Show update

### PR DESCRIPTION
**Pull request by Pouyansol**
"updated the photo, edit, delete and seach bar"

### CODE REVIEW

File: app/assets/stylesheets/components/_robots.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.card {
```
Comment (Stefan9326) [inline app/assets/stylesheets/components/_robots.scss:1]: remove this file and modify the card classes in _cards.scss

**Review by Stefan9326 — COMMENTED** [review]
make the quick changes and we can merge

**Comment by Pouyansol** [comment]
Updated

File: app/assets/stylesheets/components/_robots.scss (hunk 1 above)
Comment (Pouyansol) [reply app/assets/stylesheets/components/_robots.scss]: Resolved

**Review by Stefan9326 — APPROVED** [review]

**Pull request merged**